src/main/target/common_pre.h

```diff
@@ -124,6 +124,7 @@
 #define USE_RTC_TIME
 #define USE_PERSISTENT_MSC_RTC
 #define USE_DSHOT_CACHE_MGMT
+#define USE_LATE_TASK_STATISTICS
```

Comment:
Is there a reason to only enable this on H7?

Reply:
Just a matter of code space on unified targets. It can be enabled on individual F4 targets without issue.

Reply:
Yeah, available space, and being economical about it by not using it for not directly flight related features, is a valid point.

I guess we could start a 

```
#if (TARGET_FLASH_SIZE > 512)
```

group of features that fall under this category, in order to solve it in a consistent way that applies generically.